app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -0,0 +1,64 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Your account - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+    {%
+        with items = [
+        {
+            "link": "/",
+            "label": "Digital Marketplace"
+        }
+        ]
+    %}
+    {% include "toolkit/breadcrumb.html" %}
+    {% endwith %}
```

Comment:
We probably want more breadcrumbs on this page - one for burr dashboard and one to the brief grandparent page? Probably best to check with @ralphhawkins.